ts/store/reducers/index.ts

```diff
@@ -153,48 +157,71 @@ export function createRootReducer(
        */
       // for retro-compatibility
       // eslint-disable-next-line no-param-reassign
-      state =
-        state &&
-        ((isActionOf(logoutSuccess, action) && !action.payload.keepUserData) ||
```

Comment:
I am not sure if I'm getting this right, because _"[...] if we don't want to [...] clear something, we must first clear [...]"_ sounds like a contradiction...?

Notice that `keepUserData` has been removed because it was always set to "false" - so the case where this code was not executed does not exist

Reply:
Got it. Yes, you're right. Rephrasing would be "[...] If we don't want to persist ~~or clear something~~ [...]", but I got the point.